cmd/daprd/main.go

```diff
@@ -103,7 +105,8 @@ func main() {
 		WithBindings(bindingsLoader.DefaultRegistry).
 		WithCryptoProviders(cryptoLoader.DefaultRegistry).
 		WithHTTPMiddlewares(httpMiddlewareLoader.DefaultRegistry).
-		WithWorkflows(workflowsLoader.DefaultRegistry)
+		WithWorkflows(workflowsLoader.DefaultRegistry).
+		WithWorkflowBackends(wfbeLoader.DefaultRegistry)
```

Comment:
Just a thought:
In this PR, we have used here 3 naming conventions: `workflowbackend`, `wfbackend` and `wfbe`.
Should we make it consistent?

Reply:
I have followed existing naming convention in the codebase, wfbackend is the package name, wfbe is the variable name, where as workflowbackend is a type.

I have fixed the names at few places now to make the variable name consistent.